spotify_test_data/spotifysong-1466184460.json

```diff
@@ -0,0 +1 @@
+{"album":"Tommy Tutone - 2","artist":"Tommy Tutone","id":"6iX1f3r7oUJnMbGgQ2gx1j","title":"867-5309 / Jenny"}
```

Comment:
You could have one JSON file with a list of JSON objects in it, if that's any easier to wrangle.